components/HomepageSection/ServicesTable.jsx

```diff
@@ -0,0 +1,58 @@
+import Image from "next/image";
+import Link from "next/link";
+import services from "../../data/services.json";
+
+const ServicesTable = () => {
```

Comment:
Yes, good spot, it's not used yet but I created it in the process of designing the other component and I think it could be useful soon. Will add a comment to make this clear.